src/fides/api/util/errors.py

```diff
@@ -72,15 +72,40 @@ class ForbiddenError(HTTPException):
     To be raised when a user cannot modify an entity.
     """
 
-    def __init__(self, resource_type: str, fides_key: str) -> None:
+    def __init__(
+        self,
+        resource_type: str,
+        fides_key: str,
+        error_message: str = "user does not have permission to modify",
+    ) -> None:
         detail = {
-            "error": "user does not have permission to modify",
+            "error": error_message,
             "resource_type": resource_type,
             "fides_key": fides_key,
         }
         super().__init__(status.HTTP_403_FORBIDDEN, detail=detail)
 
 
+class ForbiddenIsDefaultTaxonomyError(ForbiddenError):
```

Comment:
Added this as an explicit way to encapsulate these types of "is default" logic errors and provide a more useful message.